parallel_cluster/conf/config

```diff
@@ -23,12 +23,40 @@ scheduler             = slurm
 initial_queue_size    = 1
 custom_ami            = ami-002b1d0f440e6caea
 
+[cluster umccr_dev]
+base_os               = alinux2
+vpc_settings          = umccr_dev_network
+# Definitely experimental
+fsx_settings          = lustrefs
+s3_read_resource      = *
+cluster_type          = spot
+# Need something substantial to hold the slurm database
+master_instance_type  = t2.medium
+# 16 CPUs and 64 Gb
+compute_instance_type = m5.4xlarge
+ec2_iam_role          = parallelcluster-ec2-instance-role
+scheduler             = slurm
+initial_queue_size    = 1
+# Basic ami with installations of
+# R, python3.8, conda, pip, ruby, golang, rust
+# conda has been initialised to the ec2-user
+custom_ami            = ami-0b98c49201c9a1e75
+
```

Comment:
Can we add `tags`?
(to avoid Config rule violations)

Reply:
This has been added in the start_cluster.sh script with the following commands:
```
      --cluster-template "${cluster_template_arg}" \
      --tags "{\"Creator\": \"${USER}\"}"
```